be/src/http/action/transaction_stream_load.cpp

```diff
@@ -421,7 +421,7 @@ void TransactionStreamLoadAction::on_chunk_data(HttpRequest* req) {
         }
         bb->pos = remove_bytes;
         bb->flip();
-        auto st = ctx->body_sink->append(bb);
+        auto st = ctx->body_sink->append(std::move(bb));
```

Comment:
should support chunked json transfer both StreamLoad & TransactionStreamLoad

Reply:
I see. Maybe it's better to create another PR to solve this.